client/src/pages/404Page/index.ts

```diff
@@ -0,0 +1 @@
+export * from './FourOhFour';
```

Comment:
I created an index.ts for all the pages and moved the pages inside a folder. This way everywhere we reference the pages we do by the index file but keep the code inside its own named file.